unlock-app/package.json

```diff
@@ -19,8 +19,7 @@
     "cross-env": "^5.2.0",
     "ethereumjs-util": "^6.0.0",
     "history": "^4.7.2",
-    "jest": "^23.6.0",
-    "jest-dom": "^3.0.0",
+    "jest-dom": "^3.1.0",
```

Comment:
are these jest-dom and terser changes intentional? PR doesn't mention them

Reply:
Yes! They are sub-dependencies which were used by jest and that we need to keep.